addons/web/static/src/core/network/rpc_cache.js

```diff
@@ -226,4 +244,9 @@ export class RPCCache {
         this.indexedDB.invalidate(tables);
         this.ramCache.invalidate(tables);
     }
+
+    cleanUpExpiredEntries() {
+        this.indexedDB.cleanUpExpiredEntries();
+        this.ramCache.cleanUpExpiredEntries();
```

Comment:
Doing this at startup is useless as the ram cache is always empty. Actually, I'm not sure we need this for the ram cache, as it is cleared on reload anyway.

Maybe the ramCache should do the setInterval itself, to clear expired entries every 30 minutes or so. That would make sense for tabs kept alive for a long time. I could then use that setInterval to check if the number of entries in the ram cache is reasonnable, and if not, clear some as well (even if not expired).